collect_app/src/test/java/org/odk/collect/android/formmanagement/FetchFormDetailsServerFormUseCasesTest.kt

```diff
@@ -308,10 +323,9 @@ class ServerFormsDetailsFetcherTest {
                 .build()
         )
 
-        val serverFormDetails = fetcher.fetchFormDetails()
+        val serverFormDetails = ServerFormUseCases.fetchFormDetails(formsRepository, formSource)
         val form = getFormFromList(serverFormDetails, "form-2")
-        assertThat(form.isUpdated, `is`(false))
-        assertThat(form.isNotOnDevice, `is`(false))
+        assertThat(form.type, equalTo(ServerFormDetails.Type.OnDevice))
```

Comment:
It would be good to update test titles here (and above) as well from `...isNotNewOrUpdated` to `...isOnDevice`.